ledger/ledger-utxo/src/main/java/net/corda/v5/ledger/utxo/query/VaultNamedQuery.java

```diff
@@ -0,0 +1,43 @@
+package net.corda.v5.ledger.utxo.query;
+
+import net.corda.v5.base.annotations.DoNotImplement;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * An interface representing a named ledger query. A named ledger query must have a name associated with it,
+ * the rest of the fields are optional.
+ */
+@DoNotImplement
+public interface VaultNamedQuery {
+
+    /**
+     * @return The name of the named ledger query
```

Comment:
Pedantic comment. Can you use `.` at the end of every sentence/single line to keep things consistent. Think its the same in other files.